pkg/util/k8scmcache.go

```diff
@@ -172,6 +172,9 @@ func (k8scm *K8sCMCache) Get(identifier string, data interface{}) error {
 func (k8scm *K8sCMCache) Delete(identifier string) error {
 	err := k8scm.Client.CoreV1().ConfigMaps(k8scm.Namespace).Delete(identifier, nil)
 	if err != nil {
+		if apierrs.IsNotFound(err) {
+			klog.V(4).Infof("k8s-cm-cache: assuming metadata configmap %s already deleted", identifier)
```

Comment:
can we return `nil` to avoid printing log at line 180 `klog.V(4).Infof("k8s-cm-cache: successfully deleted metadata configmap %s", identifier)`

@gman0 I have one question.
if user  manually deletes this entry from the config map by this changes we may end up in having stale volume at backend

Reply:
Absolutely, that's why it says it's only assuming that the volume is deleted

Reply:
And yes, I'll fix that